python/ray/monitor.py

```diff
@@ -82,10 +82,12 @@ class Monitor:
     """
 
     def __init__(self,
+                 log_dir,
```

Comment:
Would it be possible to make this optional with default "/tmp"?
That way we could avoid having to change the K8s operator (and maybe the multinode test Ameer mentioned)
I could then make a decision on how to better set this argument for the operator later in another PR.

Reply:
Let's use `None` and print to stdout, so the logs are still easily accessible while we make this decision.

Reply:
why wait for another PR?

Reply:
The decision is to not split operator logs (operator logging remains the same).
More refined logging for the operator is something to think about later.